#### Instructions

1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of `μ0 = 130.1 mm Hg` with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

   - Set up the hypothesis test.
   - Write down all the steps followed for setting up the test.
   - Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

### Hypothesis Test

**The hypothesis is that the average blood pressure of the study group is the same as the average of the general population.**
- H0: sample_mean = pop_mean (μ = μ0)  
- H1: sample_mean /= pop_mean 

Further definitions:
- Two-tailed test.
- We can use t-test or z-test.
- Level of significance alpha = 0.01 because it is data from the medical field. *The lower the α, the lower the power; the higher the α, the higher the power. The lower the α, the less likely it is that we will make an error (i.e. reject the null when it's true). The lower the α , the more “rigorous” the test.*
- Calculating the test statistic.

In [3]:
import math

sample_mean = 130.1 
pop_mean = 120
pop_std = 21.21
n = 100

statistic = (sample_mean - pop_mean)/(pop_std/math.sqrt(n))
statistic 

4.761904761904759

- Checking the critical value. <br>
dof = 99 <br>
alpha = 0.01 <br>
t = 2.626 <br>

In [4]:
from scipy.stats import t
p_value = t.sf(abs(statistic), n-1) * 2 # two-tailed
p_value

6.562701817208617e-06

**Conclusions**

The critical value from the t-test is approx. (-)2.626. <br>
Since the p-value (6.562701817208617e-06) is smaller than the significance level (0.01), we can conclude that there is sufficient evidence to reject the null hypothesis. The observed test statistic is statistically significant. <br>
In this case, the test statistic (4.761904761904759) is greater than the critical value (2.626), which also supports the rejection of the null hypothesis.
In relation to their average systolic blood pressure the group is different to the general population.